syntax = "proto3";
package QikkDB.Types;

message ComplexPolygon {
    repeated Polygon polygons = 1;
}

message Polygon {
    repeated GeoPoint geoPoints = 1;
}

message GeoPoint {
    float latitude = 1;
    float longitude = 2;
}